### ISSUE TITLE
Built-ins functions are not reserved keywords or known identifiers

### ISSUE THREAD

**Issue by anouard24** [chermehdi/comet#16]
I tried that:
```go
>> println
Comet error:

        Identifier (println) is not bounded to any value, have you tried declaring it?
>> var println = 5
CometNop
>> println
CometInt(5)
>> println("Hello, World!")
Hello, World!
CometNop
```
As a Python developer, I'm expecting that:
```go
>> println
<func builtins.println>
```
They are two options:

- Let the developer overwrite builtins functions like user's functions:
```go
>> func add(n) {2*n}
<func add(n)>
>> add = 70
CometInt(70)
```
- Make builtins functions reserved keywords

**Comment by chermehdi** [comment]
Yes, I am aware of the current issue, this hasn't been tackled yet because we don't have a module system in place, taking an action now to resolve this, might be a little bit early, as we don't know what are the requirements of the module system, and whether we can have a different solution once that has been implemented.

That being said, I will be leaving this issue open to fix it later. Thanks for pointing this out.
